tests/integration/run_plugin/provider-nodejs/index.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 Pulumi Corporation.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as provider from "@pulumi/pulumi/provider";
+
+class Provider implements provider.Provider {
+    constructor(readonly version: string) {}
+
+    async construct(name: string, type: string, inputs: pulumi.Inputs,
+        options: pulumi.ComponentResourceOptions): Promise<provider.ConstructResult> {
+	const typeName = type.split(":", 3)[2];
+        return {
+            urn: pulumi.createUrn(type, name),
+	    state: {
+		"ITS_ALIVE": "IT'S ALIVE!",
+	    }
+        }
```

Comment:
there's some indentation thing going on here, might not be visible depending on your GitHub settings
<img width="430" alt="Screenshot 2024-11-22 at 13 21 22" src="https://github.com/user-attachments/assets/538111ef-de17-4289-a7d4-6a2be9f90adb">

Reply:
Ah thanks!  I usually rely on the linter to catch this, but we're not linting these.